Resources/Prototypes/Entities/Mobs/Species/human.yml

```diff
@@ -164,9 +164,11 @@
       200: !type:DeadMobState {}
   - type: Destructible
     thresholds:
-      400:
-        behaviors:
-          - !type:GibBehavior { }
+    - trigger:
+        !type:TotalDamageTrigger
```

Comment:
While you're at it, could you make this only happen with blunt damage? (although even THAT is still placeholder, as gib shouldn't be triggered by a certain amount of damage, but...)

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3080/commits/d51ce1dd25022d29763525b12fa51c976b249072